app/models/greeting.rb

```diff
@@ -0,0 +1,2 @@
+class Greeting < ApplicationRecord
+end
```

Comment:
- [ ] Good job with your backend application, everything is working as expected. The issue is that your frontend app doesn't display any message. You can get all comments reviews you need to fix for **[the frontend repo pull request](https://github.com/williamscch/hello-react-front-end/pull/1/)**